### PR TITLE
[INLONG-2465][Manager] GetClusterConfig interface support select sinkParams

### LINKED ISSUE #2161: [Feature][InLong-Manager] Manager support getClusterConfig
### Description

[Feature][InLong-Manager] Manager support getClusterConfig 

### Use case

request URL         /api/inlong/manager/openapi/sort/standalone/getClusterConfig
requset method   GET
request params  ?clusterName=xxx&md5=xxx

example

image

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### LINKED ISSUE #2465: [Feature] GetClusterConfig interface support select sinkParams
### Description

add sinkParams table and UT
parent issue #2161 

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by imvan**
### Title Name: [INLONG-2465][Manager] GetClusterConfig interface support select sinkParams

Fixes #2465 

### Motivation

Last part of getClusterconfig interface.

### Modifications

Add sink params table which maintain the params of sort sink.

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

**Comment by dockerzhang** [comment]
@imvan please create an issue for this PR, #2161 has been fixed.

**Comment by imvan** [comment]
> @imvan please create an issue for this PR, #2161 has been fixed.

thx, has created a new issue #2465

File: inlong-manager/manager-dao/src/main/resources/generatorConfig.xml [hunk 1]
```diff
@@ -224,9 +224,15 @@
...
                enableCountByExample="false" enableDeleteByExample="false"
-               enableSelectByExample="false" enableUpdateByExample="false"/>-->
+               enableSelectByExample="false" enableUpdateByExample="false"/>
 
         <table tableName="sort_task_id_param" domainObjectName="SortTaskIdParamEntity"
+               enableSelectByPrimaryKey="true"
+               enableUpdateByPrimaryKey="true"
+               enableDeleteByPrimaryKey="true" enableInsert="true"
+               enableCountByExample="false" enableDeleteByExample="false"
+               enableSelectByExample="false" enableUpdateByExample="false"/>-->
```
Comment (luchunliang) [inline inlong-manager/manager-dao/src/main/resources/generatorConfig.xml:234]: Please remove unusable code.

**Review by wardlican — APPROVED** [review]

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/SortServiceImpl.java [hunk 2]
```diff
@@ -80,12 +83,14 @@ public SortClusterConfigResponse getClusterConfig(String clusterName, String md5
     private SortTaskConfig getTaskConfig(SortClusterConfigEntity clusterConfig) {
         List<Map<String, String>> idParams =
                 sortTaskIdParamService.selectByTaskName(clusterConfig.getTaskName());
-        // TODO add method that get sink params
+        Map<String, String> sinkParams =
+                sortTaskSinkParamService
+                        .selectByTaskNameAndType(clusterConfig.getTaskName(), clusterConfig.getSinkType());
         return SortTaskConfig.builder()
                 .taskName(clusterConfig.getTaskName())
                 .sinkType(SinkType.valueOf(clusterConfig.getSinkType().toUpperCase()))
```
Comment (luchunliang) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/SortServiceImpl.java:91]: Please filter unsupported SinkType.

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/SortTaskSinkParamServiceImpl.java [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+import java.util.Map;
+
+/** Sort sink params service implementation. */
+@Service
+public class SortTaskSinkParamServiceImpl implements SortTaskSinkParamService {
+    private static final Logger LOGGER = LoggerFactory.getLogger(SortTaskSinkParamServiceImpl.class);
+
+    private static final String KEY_SINK_TYPE = "type";
+
+    @Autowired private SortTaskSinkParamEntityMapper sortTaskSinkParamEntityMapper;
```
[the remark this answers is not in the document]
Comment (imvan) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/SortTaskSinkParamServiceImpl.java:39]: fixed

File: inlong-manager/manager-dao/src/main/resources/generatorConfig.xml (hunk 1 above)
Comment (imvan) [reply inlong-manager/manager-dao/src/main/resources/generatorConfig.xml:234]: fixed

**Review by dockerzhang — APPROVED** [review]

**Review by luchunliang — APPROVED** [review]

**Review by gosonzhang — APPROVED** [review]

**Review by healchow — APPROVED** [review]

**Pull request merged**